README.md

```diff
@@ -1 +1,8 @@
 # gluster-mixins
```

Comment:
Can you Add a bit about the project and what it's going to do?

Reply:
ack.